CODEOWNERS

```diff
@@ -875,6 +875,8 @@ build.json @home-assistant/supervisor
 /tests/components/matrix/ @PaarthShah
 /homeassistant/components/matter/ @home-assistant/matter
 /tests/components/matter/ @home-assistant/matter
+/homeassistant/components/mawaqit/ @MAWAQIT @moha-tah @Yeyvo
```

Comment:
@MAWAQIT and @Yeyvo  Are you willing to be code owners of this integration and maintain it in the future?

Reply:
Yes, I confirm

Reply:
As you both are not part or at least not publicly visible of the organization, I need a confirmation of Sagby
![grafik](https://github.com/user-attachments/assets/5741540b-28b6-43de-ba21-c0ab55e42f8b)

Reply:
Yes I confirm that Yeyvo and moha-tah are MAWAQIT organization private members.